tsconfig.json

```diff
@@ -1,13 +1,16 @@
 {
   "compilerOptions": {
     "allowJs": false,
-    "typeRoots": ["./node_modules/@types", "./src/typings"],
-    "types": ["mocha", "node"],
+    "typeRoots": ["./node_modules/@types"],
+    "types": ["mocha"],
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "module": "CommonJS",
+    "module": "ESNext",
```

Comment:
should we set the target as well?

Reply:
Not really. The code will be plyfilled later in `live-connect`.